### PR TITLE
#57 - Resolve "Sort history descendingly"

### LINKED ISSUE #57: Sort history descendingly
## Description
_**As a**_  User
_**I want**_  to have the history sorted
_**so that**_ the latest watched movie appears first

On this occasion, also let the user jump directly to specific pages in the history

## Acceptance Criteria
- [ ] Jump directly to specific pages
- [x] History is sorted with latest at top

### PR DESCRIPTION
**Pull request by JulianDroste**
Please review the changes introduced. 
The second acceptance criterium should be resolved by that.
On that note I would suggest splitting out the jumping to specific pages to a different issue.

### CODE REVIEW

File: src/history_behaviour.rs [hunk 1]
```diff
@@ -64,11 +64,12 @@ pub fn show_history(bot_data: &mut crate::BotData, order: String) {
                             .expect("Movie did not have a watched_or_removed_timestamp in show_history")
                         )
             );
-            
+            if eq(reverse,true) {
```
Comment (J4YB3) [inline src/history_behaviour.rs]:
```suggestion
            if reverse {
```

Why compare to true if boolean only has two options?

File: src/history_behaviour.rs [hunk 2]
```diff
@@ -7,7 +7,7 @@ use crate::{COLOR_BOT};
 /**
  * Formats the watch list hash map as a movie history and sends it as an embedded message
  */
-pub fn show_history(bot_data: &mut crate::BotData, order: String) {
+pub fn show_history(bot_data: &mut crate::BotData, order: String, reverse: Bool) {
```
Comment (J4YB3) [inline src/history_behaviour.rs]:
```suggestion
pub fn show_history(bot_data: &mut crate::BotData, order: String, reverse: bool) {
```

The native type is written with a lowercase `b`

**Review by J4YB3 — CHANGES_REQUESTED** [review]
Please change the two remarks 😉 Otherwise this should be fine

File: src/history_behaviour.rs (hunk 1 above)
Comment (JulianDroste) [reply src/history_behaviour.rs]: was not aware that rust can do that too 👍🏼

**Review by J4YB3 — APPROVED** [review]

**Pull request merged**